src/logger.js

```diff
@@ -2,7 +2,20 @@
 // Logger
 //
 
-const winston = require('winston')
+const { createLogger, format, transports } = require('winston');
+const { combine, timestamp, printf, splat, colorize } = format;
 
-winston.handleExceptions(new winston.transports.Console({ colorize: true }))
-module.exports = winston
+const myFormat = printf(({ level, message, timestamp }) => {
+  return `${timestamp} ${level}: ${message}`;
+});
+
+module.exports = createLogger({
+  level: 'info',
+  format: combine(
+    splat(),
+    colorize(),
+    timestamp(),
+    myFormat
+  ),
+  transports: [new transports.Console()]
+});
```

Comment:
Is it producing much more logs that can be sensitive?

Reply:
Please check the log behavior.

Reply:
This change was done only to support breaking changes in new winston logger version.
Previous implementation worked the same:
```winston.handleExceptions(new winston.transports.Console({ colorize: true }))```

Reply:
Logs contains job name, labels and other properties (but not the task properties):
```
{"kind":"CronJob","metadata":{"name":"d438967c-4a5a-409d-a448-106e026e2d3a","labels":{"id0":"Zdmp-ea9ee84b-4a90-4a42-9f57-aa45c4e15f64-tasks-tZ","id1":"Zask_schedule1Z","tenant":"Zea9ee84b-4a90-4a42-9f57-aa45c4e15f64Z","type":"Zdmp-schedulingZ","taskId":"Ztask_schedule1Z","recurrence":"ZfalseZ"},"finalizers":["foregroundDeletion"]},"spec":{"schedule":"0 12 20 11 ?","successfulJobsHistoryLimit":1,"failedJobsHistoryLimit":1,"jobTemplate":{"spec":{"template":{"spec":{"enableServiceLinks":false,"containers":[{"name":"executor","image":"mmp2devwestus2registry.azurecr.io/dmp-scheduler-executor@sha256:25cef6769334430fe40270e48078835e1c688b2fa5df94007810bc85a1d7ecca","args":[],"imagePullPolicy":"IfNotPresent","env":[{"name":"JOB","value":"{\"name\":\"dmp-ea9ee84b-4a90-4a42-9f57-aa45c4e15f64-tasks-task_schedule1\",\"type\":\"rest\",\"namespace\":\"vlavrenc\",\"request\":{\"uri\":\"http://vlavrenc-internal-proxy/api/tasks/task_schedule1\",\"method\":\"PATCH\",\"json\":true,\"headers\":{\"Tenant\":\"ea9ee84b-4a90-4a42-9f57-aa45c4e15f64\"},\"body\":{\"action\":\"Start\",\"status\":\"In Progress\",\"lastResultDescription\":\"Start on schedule\"}},\"recur\":{\"start\":null,\"triggers\":[\"0 12 20 11 ?\"],\"end\":null},\"labels\":{\"tenant\":\"ea9ee84b-4a90-4a42-9f57-aa45c4e15f64\",\"type\":\"dmp-scheduling\",\"taskId\":\"task_schedule1\",\"recurrence\":\"false\"}}"}]}],"restartPolicy":"OnFailure"}}}}}}
```